styles.css

```diff
@@ -0,0 +1,63 @@
+header {
+  display: flex;
+  align-items: flex-end;
+  flex-direction: column;
+}
+
+#mobile-navbar img {
+  width: 50px;
+}
+
+#background-image {
+  background-image: url("./assets/header-bg-mobile.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+  height: 100vh;
+}
```

Comment:
- [ ] Please use Flexbox to align the content of this section. Because it gives us a lot of options to align all our elements, plus, it is responsive 😉😉😉

- [ ] [OPTIONAL] I advise you to rename this id to something more descriptive of this section, for example, you can use <code>headline</code>